shards/modules/core/core.hpp

```diff
@@ -1221,11 +1221,7 @@ struct Ref : public SetBase {
 
   ALWAYS_INLINE SHVar activateRegular(SHContext *context, const SHVar &input) {
     // must keep flags!
-    const auto rc = _target->refcount;
-    const auto flags = _target->flags;
-    memcpy(_target, &input, sizeof(SHVar));
-    _target->refcount = rc;
-    _target->flags = flags;
+    assignVariableValue(*_target, input);
```

Comment:
This is not gonna work also I don't want a redirection here, it's supposed to be as bare as possible on purpose.